bengoshow/unit-converter
# feat: store price changes in state

**Summary**
This change moves collection data into state, tracking changes to item price in the state object.

**Validation**
1. Visit the current site at https://unit-converter-jet-chi.vercel.app/
2. Select a collection from the drop-down
3. Add or alter a price for an item
4. Select a different collection
5. Return to your original collection and notice the price changes are lost

<hr />

1. Visit this PR branch at https://unit-converter-git-track-collections-state-change-bengoshow.vercel.app/
2. Select a collection from the drop-down
3. Add or alter a price for an item
4. Select a different collection
5. Return to your original collection and confirm the price changes remain
